examples/expe_conv_enet/main_enet.py

```diff
@@ -9,20 +9,28 @@
 from andersoncd.lasso import solver_enet, apcg
 
 
-dataset_names = [
-    "leukemia", "gina_agnostic", "hiva_agnostic", 'rcv1_train', 'news20']
+dataset_names = ["leukemia"]
+# dataset_names = [
+#     "leukemia", "gina_agnostic", "hiva_agnostic", 'rcv1_train', 'news20']
 
 
+div_alphas = [10, 100, 1000, 5_000]
+div_rhos = [10, 100]
+
+
+n_jobs = 1
+# n_jobs = len(dataset_names) * len(div_alphas) * len(algos)
```

Comment:
rm

Reply:
why? I just set n_jobs to 1 to be sure that someone not experienced with the script won't launch it with 15 jobs, but it seems logical to let it no?